Updated note about M10 GPSes (#512)
betaflight/betaflight.com: docs/wiki/getting-started/hardware/index.md

:::danger

> Most GPS units on the market are based on the M8 chipset from uBlox. However, there are newer modules that use the M10 chipset. These work much better than M8... with a slight catch. Prior to the 4.5.0 release, the M10 was not supported for auto-configuration in Betaflight, so you had to configure it manually in the u-center software. If not configured properly, it lead to unreliable performance. On releases after 4.5.0, the manual configuration is no longer necessary.

Reviewer: ```suggestion
Most GPS units on the market are based on the M8 chipset from uBlox. However, there are newer modules that use the M10 chipset. These work much better than M8... with a slight catch. Prior to the 4.5.0 release, the M10 was not supported for auto-configuration in Betaflight, so you had to configure it manually in the u-center software. If not configured properly, this will result in unreliable performance. From release 4.5.0, manual configuration is no longer necessary.
```

> Most GPS units on the market are based on the M8 chipset from uBlox. However, there are newer modules that use the M10 chipset. These work much better than M8... with a slight catch. Prior to the 4.5.0 release, the M10 was not supported for auto-configuration in Betaflight, so you had to configure it manually in the u-center software. If not configured properly, this will result in unreliable performance. From release 4.5.0, manual configuration is no longer necessary.

Reviewer 2: ```suggestion
Most GPS units on the market are based on the M8 chipset from uBlox. However, there are newer modules that use the M10 chipset. These work much better than M8... with a slight catch. Prior to the 4.5.0 release, the M10 was not supported for auto-configuration in Betaflight, As such, it required manual configuration in the u-center software. When not properly configured, this results in unreliable performance. From release 4.5.0, manual configuration is no longer necessary.
```
